packages/manager/apps/container/src/container/nav-reshuffle/header/user-account-menu/DefaultPaymentMethod.tsx

```diff
@@ -17,69 +13,27 @@ const UserDefaultPaymentMethod = ({
   isLoading = false,
 }: Props): JSX.Element => {
   const { t } = useTranslation('user-account-menu');
-  const shell = useShell();
-  const trackingPlugin = shell.getPlugin('tracking');
-  const { closeAccountSidebar } = useProductNavReshuffle();
-
-  const onPaymentMethodLinkClick = () => {
-    closeAccountSidebar();
-    trackingPlugin.trackClick({
-      name: 'topnav::user_widget::go_to_payment_method',
-      type: 'navigation',
-    });
-  };
-
-  // @todo: use navigation plugin instead
-  const paymentMethodUrl = useShell()
-    .getPlugin('navigation')
-    .getURL('dedicated', '#/billing/payment/method');
 
   return (
     <div
       className={`${style.defaultPaymentMethod} my-1`}
       id="user-account-menu-payment-method"
     >
-      <a
-        onClick={onPaymentMethodLinkClick}
-        className="d-flex flex-row align-items-center p-2"
-        href={paymentMethodUrl}
-        target="_top"
-      >
-        <div className="m-auto p-1 minw-0 w-100">
-          <h2>{t('user_account_menu_payment_method_title')}</h2>
-          {isLoading && (
-            <div className={`${style.defaultPaymentMethod_loading}`}>
-              <div className="oui-skeleton oui-skeleton_s">
-                <div className="oui-skeleton__loader"></div>
-              </div>
-              <div className="oui-skeleton oui-skeleton_s">
-                <div className="oui-skeleton__loader"></div>
-              </div>
-            </div>
-          )}
-          {!isLoading && defaultPaymentMethod && (
-            <div className={`${style.defaultPaymentMethod_label}`}>
-              <p className="mb-1 text-truncate">{defaultPaymentMethod.label}</p>
-              <span
-                className={`${
-                  style.defaultPaymentMethod_status
-                } oui-badge oui-badge_${defaultPaymentMethod.getStatusCategory()}`}
-              >
-                {t(
-                  `user_account_menu_payment_method_status_${defaultPaymentMethod.status}`,
-                )}
-              </span>
-            </div>
-          )}
-          {!isLoading && !defaultPaymentMethod && (
-            <p>{t('user_account_menu_payment_method_none')}</p>
-          )}
+      {!isLoading && defaultPaymentMethod && (
+        <div className="d-flex mt-1 justify-content-between">
+          <span>{t('user_account_menu_payment_method_title')}</span>
+
+          <span
```

Comment:
I thought we said we would redirect to the payment page on clicking on the badge

Reply:
I don't see anything about it on the ticket or specs.
When was this mentionned ?

Reply:
It is defined here: https://confluence.ovhcloud.tools/pages/viewpage.action?pageId=375164986#Specs[WIP]-userWidget
(Specs link present in the ticket)